smithy-typescript-integ-tests/codegen/build.gradle

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *   http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+import software.amazon.smithy.gradle.tasks.SmithyBuild
+
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'software.amazon.smithy:smithy-typescript-codegen:0.3.0'
+        classpath 'software.amazon.smithy:smithy-aws-typescript-codegen:0.3.0'
```

Comment:
SSDK generation does not work at the moment without a protocol selected, which is technically something to fix, but then again, what is the (current) practical use of SSDK generation without a protocol?

Reply:
Practically not much except as a hack to only get types. But it could mean that we've got some stuff buried in protocol implementations that shouldn't be

Reply:
No protocol:

```
Unable to find a protocol generator for software.amazon.smithy.typescript.integ#ValidationService: Unable to derive the protocol setting of the service `software.amazon.smithy.typescript.integ#ValidationService` because no protocol definition traits were present. You need to set an explicit `protocol` to generate in smithy-build.json to generate this service.

Projection ts-server failed: java.lang.NullPointerException
software.amazon.smithy.typescript.codegen.ServerCommandGenerator.lambda$writeOperationSerializer$0(ServerCommandGenerator.java:153)
...
```

No protocol and no service:

```
Projection ts-server failed: software.amazon.smithy.codegen.core.CodegenException: Cannot infer a service to generate because the model does not contain any service shapes
software.amazon.smithy.typescript.codegen.TypeScriptSettings.inferService(TypeScriptSettings.java:104)
...
```